docs/snippets/rename-a-country.md

```diff
@@ -0,0 +1,22 @@
+# Rename a country
+
+> This is a **Developer level** doc. If you are unfamiliar with code and resolving potential conflicts, select a [WooExpert or Developer](https://woocommerce.com/customizations/) for assistance. We are unable to provide support for customizations under our [Support Policy](http://www.woocommerce.com/support-policy/).
+
+Add this code to your child theme’s `functions.php` file or via a plugin that allows custom functions to be added, such as the [Code Snippets](https://wordpress.org/plugins/code-snippets/) plugin. Avoid adding custom code directly to your parent theme’s functions.php file, as this will be wiped entirely when you update the theme.
+
+```php
+if ( ! function_exists( 'YOUR_PREFIX_rename_country' ) ) {
+  /**
+   * Rename a country
+   *
+   * @param array $countries Existing country names
+   * @return array $countries Updated country name(s)
+   */
+  function YOUR_PREFIX_rename_country( $countries ) {
+     $countries['IE'] = __( 'Ireland', 'YOUR-TEXTDOMAIN' ),
```

Comment:
Needs a small fix to prevent a fatal error:

```suggestion
     $countries['IE'] = __( 'Ireland', 'YOUR-TEXTDOMAIN' );
```

Also, and I realize we're chiefly interested in copying existing docs across, rather than finessing them, but I wonder if something like this would be clearer:

```suggestion
     $countries['IE'] = __( 'Ireland (Changed)', 'YOUR-TEXTDOMAIN' );
```

Otherwise, since Ireland renders as Ireland by default (at least in English), a user testing this out wouldn't see any change. This small tweak might make things a little more obvious, though.

(Anyway, this second bit is just an optional thing to consider, we definitely don't need it.)